### DISCUSSION TITLE
Traccar

### DISCUSSION THREAD

**Discussion by juhacz** [runtipi/runtipi-appstore#3222] [Requests]
### Description

Modern GPS Tracking Platform

### Official Website

https://www.traccar.org/

### Documentation link

https://www.traccar.org/documentation/

### Last application release & date

6.1

### Application license

Apache License, Version 2.0

### Source code repository link

https://github.com/traccar/traccar

### Docker image link

https://hub.docker.com/r/traccar/traccar

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by Scrub000** [comment]
This would be perfect
